### PR TITLE
[validation] validation strategy

### LINKED ISSUE #857: Create strategy for ticket-multiple-validity
**Logic:** **If the user has at least `1` voting power in any one strategy, return `1` else return `0`**

- Should be similar to https://github.com/snapshot-labs/snapshot-strategies/tree/master/src/strategies/ticket-validity
- Add a way to pass multiple strategies `options.strategies` , maximum of `3` (for example to pass `erc20-balance-of` and `contract-call` strategies)
- `options.min` - to check the minimum voting power in one strategy (Default to `0`)

### PR DESCRIPTION
**Pull request by ChaituVR**
Fixes #857

### CODE REVIEW

File: src/strategies/validation/index.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      )
+    );
+  }
+
+  const results = await Promise.all(promises);
+  let validatedAddresses: string[] = [];
+  
+  results.forEach((result) => {
+    for (const address in result) {
+      if (result[address] > minimumValidity) {
```
Comment (ChaituVR) [inline src/strategies/validation/index.ts:47]:
```suggestion
      if (result[address] >= minimumValidity) {
```

File: src/strategies/validation/examples.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          },
+          {
+            "name": "erc20-balance-of",
+            "params": {
+              "address": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+              "decimals": 6
+            }
+          }
+        ],
+        "minimumValidity": 0,
```
Comment (ChaituVR) [inline src/strategies/validation/examples.json:24]:
```suggestion
        "minimumValidity": 1,
```

File: src/strategies/validation/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+{
+    "symbol": "UNI",
+    "validationStrategies": [{
+        "name": "erc20-balance-of",
+        "params": {
+            "address": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+            "decimals": 6
+        }
+    }],
+    "minimumValidity": 0,
```
Comment (ChaituVR) [inline src/strategies/validation/README.md:26]:
```suggestion
    "minimumValidity": 1,
```

**Review by bonustrack — APPROVED** [review]
utACK

**Pull request merged**